lib/sms/sms_submit.c

```diff
@@ -88,6 +88,32 @@ static int sms_submit_encode_number(
 	return 0;
 }
 
+/**
+ * @brief Prints error information for positive error codes of @c nrf_modem_at_printf.
+ *
+ * @param[in] err Error code.
+ */
+static void sms_submit_print_error(int err)
+{
+#if (CONFIG_SMS_LOG_LEVEL >= 1)
```

Comment:
I'd consider using a symbol instead of a magic number here (`LOG_LEVEL_ERR`)

Reply:
Fixed.